docs/CLI_Reference/oras_blob_fetch.mdx

```diff
@@ -49,6 +49,14 @@ Fetch and print a blob from OCI image layout archive file 'layout.tar':
 oras blob fetch --oci-layout --output - layout.tar@sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5
 ```
 
+Fetch a blob with a specific reference, determine its size using jq, and use pv to display the progress while redirecting the output to a file named layer.bin
```

Comment:
```suggestion
Fetch a blob with a specific reference, determine its size using `jq`, and use `pv` to display the progress while redirecting the output to a file named `layer.bin`:
```
I think including jq, pv. and layer.bin in code line will highlight it more. Just a nitpick. Everything else looks good.